Update/wata_007.txt

```diff
@@ -3035,10 +3036,10 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 6, "ps3/s19/06/500600491", 128, TRUE);
 	OutputLine(NULL, "「だから当時、年寄り連中はみんな言ったんです。",
-		   NULL, "\"So at the time, all the older people were like:", Line_WaitForInput);
+		   NULL, "So at the time, all the older people were like:", Line_WaitForInput);
 	ModPlayVoiceLS(4, 6, "ps3/s19/06/500600492", 128, TRUE);
 	OutputLine(NULL, "これはオヤシロさまの祟りだ、",
-		   NULL, " This is Oyashiro-sama's curse.", Line_Continue);
+		   NULL, " 'This is Oyashiro-sama's curse.'", Line_Continue);
```

Comment:
This is a quote and should be within quotes

Reply:
A quote within quote uses single marks. It's because it's a huge monologue from Shion.